.changeset/weak-poems-explode.md

```diff
@@ -0,0 +1,7 @@
+---
+'nextra': patch
+'nextra-theme-blog': patch
+'nextra-theme-docs': patch
+---
+
+remove `font-weight: 500;` from styles of code blocks since it gives no effect
```

Comment:
Hope I didn’t confuse you guys with issue #2172.

The style `font-weight: 500;` *does take effect* in those browsers that support the generic family name `ui-monospace`, such as Safari (Mac/iOS). However, in browsers that don’t support the `ui-monospace` (Chrome, Firefox, etc.), the styles `font-weight: 500;` in combination with `font-family: …, SFMono-Regular, Menlo, …;` make no sense.

Hence, I’m not sure removing `font-weight: 500;` complies with the design/typographic spec of the Doc theme. The SF Mono family does include the 500 weight. (Obviously, fonts such as Menlo, Monaco, and Consolas also do not have a font weight of 500.)

Btw, a mistake I made in #2172 was that actually the PostScript name `SFMono-Regular` seems to have no effect for the Chrome/Firefox/Safari (at least on macOS 12.6+).